content/docs/Configuration/jobs.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Packit Service jobs configuration 
+date: 2023-07-03
+weight: 1
+---
+
+# Packit Service jobs configuration
+
+Packit Service doesn't have any web interface to configure it,
+so the only way to change its behaviour is via the config file generally described [here](/docs/configuration).
+
+When you open a pull request against your upstream repository, packit service
+picks up configuration file from your pull request, not from the branch against
+the PR is opened. This way, you can polish your .packit.yaml and see the
+results right away. (for more info, please see [packit-service#48](https://github.com/packit/packit-service/issues/48))
```

Comment:
I would mention that it works like that in general -- that it takes the config from the relevant git ref (release tag for release, PR head for PRs, related dist-git commit for downstream jobs...).

Reply:
I rewrote this section that was written few years ago, PTAL